web/src/stream_popover.js

```diff
@@ -267,6 +289,7 @@ export function build_move_topic_to_stream_popover(
         const move_limit_buffer = 5;
         args.disable_topic_input = !message_edit.is_topic_editable(message, move_limit_buffer);
         disable_stream_input = !message_edit.is_stream_editable(message, move_limit_buffer);
+        args.message_type = await get_message_type(current_stream_id, topic_name, message.id);
```

Comment:
Let's call this parameter `args.message_placement`

Reply:
Fixed. 👍🏻